examples/esp/Cargo.toml

```diff
@@ -0,0 +1,69 @@
+[package]
+name = "esp-openthread-examples"
+version = "0.1.0"
+edition = "2021"
+authors = ["bjoernQ <bjoern.quentin@mobile-j.de>"]
+resolver = "2"
+categories = ["embedded", "hardware-support"]
+keywords = ["thread", "openthread", "embedded", "embassy"]
+description = "openthread examples for Espressif chips"
+repository = "https://github.com/ivmarkov/esp-openthread"
+license = "MIT OR Apache-2.0"
+readme = "README.md"
+rust-version = "1.84"
+
+[profile.dev]
+# Rust debug is too slow.
+# For debug builds always builds with some optimization
+opt-level = "z"
+
+[profile.release]
+codegen-units = 1 # LLVM can perform better optimizations using a single thread
+debug = 2
+debug-assertions = false
+opt-level = "s"
+overflow-checks = false
+
+[features]
+default = ["esp32c6"]
+esp32c6 = [ "esp-hal-embassy/esp32c6", "esp-ieee802154/esp32c6", "esp-backtrace/esp32c6", "esp-println/esp32c6"]
+esp32h2 = [ "esp-hal-embassy/esp32h2", "esp-ieee802154/esp32h2", "esp-backtrace/esp32h2", "esp-println/esp32h2"]
+
+[dependencies]
+embassy-executor = { version = "0.7", features = ["task-arena-size-32768"] }
+embassy-sync = "0.6"
+embassy-futures = "0.1"
+embassy-time = "0.4"
+embassy-net = { version = "0.6", features = ["proto-ipv6", "medium-ip", "udp"] }
+esp-hal = { version = "0.23.1", features = ["log"] }
```

Comment:
lets try to upgrade to 1.0.0-beta.0 etc. after merging

Reply:
Yup. But maybe after merge. Still I'll keep this open, if we decide otherwise.